gradle/libs.versions.toml

```diff
@@ -0,0 +1,64 @@
+[versions]
+agp = "7.1.0"
+autoService = "1.0.1"
+autoValue = "1.10"
+dagger = "2.39.1"
+espresso = "3.5.1"
+kotlin = "1.8.0"
+ktlint = "0.41.0"
+
+# Configs that we can override in CI. Not exactly "versions" but useful to repurpose it here.
+config-fullTestRun = "true"
```

Comment:
I feel like this chunk of the previous comment from dependencies.gradle is worth preserving:
```
We don't need to run all tests for sanity checks. If the flag is
false, then we run each test with one annotation instead of all options. We also skip tests
that run the Dagger annotation processor (KAPT is slow).
```